apigentools/cli.py

```diff
@@ -257,12 +252,28 @@ def get_cli_parser():
         action="store_true",
         default=False,
     )
+    push_parser.add_argument(
+        '--push-commit-msg',
+        help='Message to use for the commit when pushing the auto generated clients',
+        default=env_or_val("APIGENTOOLS_COMMIT_MSG", '')
+    )
     push_parser.add_argument(
         "--skip-if-no-changes",
         help="Skip committing/pushing for all repositories where only .apigentools-info has changed",
         action="store_true",
         default=env_or_val("APIGENTOOLS_SKIP_IF_NO_CHANGES", False, __type=bool),
     )
+    push_parser.add_argument(
+        "--git-email",
+        help="Email of the user to author git commits as",
```

Comment:
We should add a note about this setting being permanent for the repo (and do the same for the `--git-name` arg).

Reply:
done!